src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -1,51 +1,89 @@
-import React, { useCallback, useEffect, useState } from "react";
-import classNames from "classnames";
+import React, { useEffect } from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faTimesCircle } from "@fortawesome/free-solid-svg-icons";
+import classNames from "classnames";
 
 import { getLinkFromText } from "utils/getLinkFromText";
+import { isDefined } from "utils/types";
+
+import { useShowHide } from "hooks/useShowHide";
 
-import { useChatSidebarControls } from "hooks/chatSidebar";
+import { Banner } from "types/banner";
```

Comment:
can we have smth like https://github.com/lydell/eslint-plugin-simple-import-sort/ added to our dependencies?
just so we have this automated

Reply:
@alexey-petrov there is already a backlog issue about this on the internal board. When there is actual time/capacity for QoL things rather than triaging fires and urgent client features/needs, it’s at the top of the list.

Reply:
Not sure if that specific plug-in is listed there. If not, please add it

Reply:
didn't add this plugin since we have a task with a similar one in the backlog, thanks
formatted order manually 1c2d835

Reply:
@alexey-petrov i was asking you to add the link for that plug-in to that backlog issue..